language-plutus-core/stdlib/Language/PlutusCore/StdLib/Data/Function.hs

```diff
@@ -278,21 +290,22 @@ fixN n = runQuote $ do
 
     -- [A1, B1, ..., An, Bn]
     let allAsBs = foldMap (\(a, b) -> [a, b]) asbs
-    pure $
-        -- abstract out all the As and Bs
-        mkIterTyAbs (fmap (\tn -> TyVarDecl () tn (Type ())) allAsBs) $
-        lamAbs () f fTy $
-        mkIterApp () instantiatedFix
-            [ lamAbs () k kTy $
-              tyAbs () s (Type ()) $
-              lamAbs () h hTy $
-              mkIterApp () (var () h) branches
-            , var () f
-            ]
+    let fixNTerm =
+          -- abstract out all the As and Bs
+          mkIterTyAbs (fmap (\tn -> TyVarDecl () tn (Type ())) allAsBs) $
+          lamAbs () f fTy $
+          mkIterApp () instantiatedFix
+              [ lamAbs () k kTy $
+                tyAbs () s (Type ()) $
+                lamAbs () h hTy $
+                mkIterApp () (var () h) branches
+              , var () f
+              ]
+    pure (fixNTerm, fixNType)
 
 -- | Get the fixed-point of a list of mutually recursive functions.
 --
--- > MutualFixOf _ [ FunctionDef _ fN1 (FunctionType _ a1 b1) f1
+-- > MutualFixOf _ fixN [ FunctionDef _ fN1 (FunctionType _ a1 b1) f1
 -- >                         , ...
 -- >                         , FunctionDef _ fNn (FunctionType _ an bn) fn
 -- >                         ] =
```

Comment:
Since you're here, please also fix the formatting, so that `[` and `,` align properly.